drivers/Ofek.py

```diff
@@ -0,0 +1,8 @@
+from rose.common import obstacles, actions  # NOQA
+
+driver_name = "Sidan"
+
+
```

Comment:
maybe remove the spaces? it will look better.

Reply:
no. I don't like your idea.

Reply:
I respect your opinion, however you are wrong.